src/css/controls/imports/shortcuts-tooltip.less

```diff
@@ -38,41 +39,103 @@
         display: flex;
         flex: 1 1 auto;
         flex-flow: column;
+        font-size: 12px;
         margin: 0 20px 20px;
         overflow-y: auto;
         padding: 5px;
-    }
 
-    .jw-shortcuts-title {
-        font-size: 12px;
-        font-weight: bold;
-    }
+        .jw-shortcuts-title {
+            font-weight: bold;
+        }
 
-    .jw-shortcuts-tooltip-list {
-        display: flex;
-        max-width: 340px;
-        font-size: 12px;
-        margin: 0 10px;
+        .jw-shortcuts-switch {
```

Comment:
this should be jw-switch, and should be in shared imports for components. We can re-use this.